flux/linkerd/helmrelease.yaml

```
@@ -85,20 +85,17 @@ spec:
       issuer:
         scheme: kubernetes.io/tls
     disableHeartBeat: true
-    controllerImage: altinncr.azurecr.io/linkerd/controller
+    controllerImage: altinncr.azurecr.io/ghcr.io/linkerd/controller
     debugContainer:
       image:
-        name: altinncr.azurecr.io/linkerd/debug
+        name: altinncr.azurecr.io/ghcr.io/linkerd/debug
     policyController:
       image:
-        name: altinncr.azurecr.io/linkerd/policy-controller
+        name: altinncr.azurecr.io/ghcr.io/linkerd/policy-controller
     proxy:
       image:
         name: altinncr.azurecr.io/linkerd/proxy
```

Use same cache path here?
```suggestion
        name: altinncr.azurecr.io/ghcr.io/linkerd/proxy
```

---

Yes, I dont know how I missed it. Thanks.